code/game/objects/effects/spawners/random_spawners.dm

```diff
@@ -9,6 +9,8 @@
 	/obj/effect/decal/cleanable/blood/oil = 1,
 	/obj/effect/decal/cleanable/fungus = 1)
 	var/spawn_inside = null
+	/// Defines whether the spawner is deleted during its New() proc. Set to FALSE if it needs to reach Initialize().
+	var/early_del = TRUE
```

Comment:
**Please** do not do this. `/New()` is going to be butchered, and this is going to cause issues with `SSatoms` down the line if atoms are being deleted before they are properly initialized, as it will still be marked to do so